Blueprints/BlueprintDefinitions/vs2022/PowertoolsServerless/template/src/BlueprintBaseName.1/aws-lambda-tools-defaults.json

```diff
@@ -0,0 +1,19 @@
+{
+  "Information": [
+    "This file provides default values for the deployment wizard inside Visual Studio and the AWS Lambda commands added to the .NET Core CLI.",
+    "To learn more about the Lambda commands with the .NET Core CLI execute the following command at the command line in the project root directory.",
+    "dotnet lambda help",
+    "All the command line options for the Lambda command can be specified in this file."
+  ],
+  "profile": "DefaultProfile",
+  "region": "DefaultRegion",
+  "configuration": "Release",
+  "s3-prefix": "BlueprintBaseName.1/",
+  "template": "serverless.template",
+  "function-architecture": "x86_64",
```

Comment:
Can you remove the `function-` properties. They are used here because you are deploying with the CloudFormation template defined by the `serverless.template` that sets all of these settings.